tests/subsys/disk/stm32_sdmmc/boards/stm32l496g_disco.overlay

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2020 Amarula Solutions
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	sdmmc1 {
```

Comment:
Should we also add sdmmc1 entries to the `.dtsi` files? That would make it easy to add support for sdmmc1 in board `.dts` files (don't need overlay files).

Reply:
Yes, if it's tested on the board I think it would probably be right to add support in the board dts instead of overlay.

Reply:
+1 to move things to .dtsi/.dts when possible.
A node should be defined in .dtsi (undet dts/arm/st/...), then edited in board .dts for customization.
Overlay files should be used for application specific purpose when needed